src/main/java/goodgenerator/blocks/tileEntity/NeutronActivator.java

```diff
@@ -153,6 +190,7 @@ public void loadNBTData(NBTTagCompound aNBT) {
         mCeil = aNBT.getInteger("mCeil");
         mFloor = aNBT.getInteger("mFloor");
         height = aNBT.getInteger("height");
+        mBatchMode = aNBT.getBoolean("mBatchMode");
```

Comment:
Please use the variables that are now in the multiblock class. `batchMode` and `BATCH_MODE_NBT_KEY`. Makes future GUI transitions easier.

Reply:
done